packages/components/CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Change Log - @itwin/presentation-components
 
+## 5.0.0-dev.0
+
```

Comment:
Does `changesets` provide a way to add some extra description here?

Reply:
I didn't find anything about adding additional text to CHANGELOG.md but I guess you could add some text manually and it shouldn't be stripped out. I don't think CHANGELOG.md is updated after this PR is merged. Only when new changesets are pushed to base branch.

Reply:
```suggestion

The `5.0` release is targeted towards getting instance filtering production-ready and contains a number of bug fixes and enhancements for the `PresentationInstanceFilterDialog` component. 

The release does not contain any breaking API changes and the bump in peer-depenendecies is the only reason this is a major release.

```